package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "shoping-list",
+  "version": "1.0.0",
+  "description": "shoping-list",
+  "main": "app.js",
+  "scripts": {
+    "test": "jest ./tests/test.js",
+    "lint": "eslint ./app.js",
```

Comment:
why just `app.js`? I would rather use eslint ./

Reply:
you are right, changed to ./